packages/layouts/src/classes/galleryItem.js

```diff
@@ -691,22 +690,14 @@ class GalleryItem {
     );
   }
 
-  get calculatedAlt() {
-    return (
-      (utils.isMeaningfulString(this.alt) && this.alt) ||
-      this.title ||
-      this.description ||
-      this.fileName ||
-      ''
-    );
-  }
-
   get htmlContent() {
     return this.html?.replace(/<[^<>]*>/g, '').trim();
   }
 
   get alt() {
-    return this.metadata.alt || '';
+    return (
+      (utils.isMeaningfulString(this.metadata.alt) && this.metadata.alt) || ''
```

Comment:
I think it should be 
(this.metadata.alt && utils.isMeaningfulString(this.metadata.alt)) || ''

Reply:
It will return 'true' instead of the alt